modules/ROOT/pages/scheduler-xml-reference.adoc

```diff
@@ -9,6 +9,31 @@ A XML for the Scheduler has these elements:
 * `scheduling-strategy` block inside `scheduler`.
 * `fixed-frequency` or `cron` element inside the `scheduling-strategy` block.
 
+== Properties of Scheduler (`<scheduler>`)
+[%header,cols="34,33,33"]
+|===
+|Attribute |Description |Example
+|`disallowedConcurrentExecution`
+|Triggers next scheduler only if the last execution ended.
```

Comment:
It should be clear that next attempt to execute will be after another period. It won't be triggered just after the last execution ended (this only skips the attempt to execute)

Reply:
Does this sound better? `Skips the next scheduled flow execution if the last one has not ended`

Reply:
👍